events/sinks/riemann/driver.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package riemann
+
+import (
+	"net/url"
+	"sync"
+
+	"strings"
+	"time"
+
+	riemann_api "github.com/bigdatadev/goryman"
+	"github.com/golang/glog"
+	riemannCommon "k8s.io/heapster/common/riemann"
+	core "k8s.io/heapster/events/core"
+	kube_api "k8s.io/kubernetes/pkg/api"
+)
+
+// Abstracted for testing: this package works against any client that obeys the
+// interface contract exposed by the goryman Riemann client
```

Comment:
This comment doesn't make much sense here if we don't keep the interface we had before.